src/components/types.ts

```diff
@@ -28,6 +28,7 @@ import { SidebarCollapsibleItemProps } from "./SidebarCollapsibleItem/SidebarCol
 import { SidebarCollapsibleTitleProps } from "./SidebarCollapsibleTitle/SidebarCollapsibleTitle";
 
 export type States = "default" | "active" | "disabled" | "error" | "hover";
+export type IconDir = "start" | "end";
```

Comment:
we could change the type to `HorizontalDirection` since it can apply to multiple elements